### PR TITLE
Usage Instructions improvements

### PR DESCRIPTION
**Pull request by brunobowden**
- line wraps
- ‘suggested’ rather than ‘preferred’
- more ascii art
- add j2objcConfig example

### CODE REVIEW

**Comment by brunobowden** [comment]
@confile - please take a look and mark the section that you found confusing

@advayDev1 - FYI

**Comment by advayDev1** [comment]
LGTM, but it is already clear to me.

**Comment by brunobowden** [comment]
@confile - please take a look

File: README.md [hunk 1]
```diff
@@ -2,14 +2,17 @@
...
 
 ### Usage
-At HEAD, this plugin is in a state of significant flux as we refactor it into a first-class Gradle plugin for our beta.
-You may wish to wait for the beta release as we may make backwards incompatible changes before that point.
+At HEAD, this plugin is in a state of significant flux as we refactor it into a first-class
+Gradle plugin for our beta. You may wish to wait for the beta release as we may make backwards
+incompatible changes before that point.
 
-You should start with a clean java only project without any android dependencies, preferably named `'shared'`. This must be buildable using Gradle's standard `'java'` plugin. It may start as an empty project and allows you to gradually shift over code from an existing Android application.
+You should start with a clean java only project without any android dependencies. It suggested that
```
Comment (confile) [inline README.md]: This is not very clear to me. You may give a diagram of the project structure here as well.

Comment (confile) [inline README.md]: Or you could bring the project structure diagram from below much earlier so the reader get an understanding of what you meant.

Comment (brunobowden) [inline README.md]: Added link to the project structure. If you have alternative text that you'd like to propose, then I'd be happy to look at a PR @confile.

**Comment by confile** [comment]
LGTM @brunobowden great job

**Pull request merged**